src/Campaigns/Blocks/CampaignTitleBlock/block.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "https://schemas.wp.org/trunk/block.json",
+    "apiVersion": 3,
+    "name": "givewp/campaign-title-block",
+    "version": "1.0.0",
+    "title": "Campaign Title",
+    "category": "give",
+    "icon": "heading",
+    "description": "Displays the title of the campaign.",
+    "attributes": {
+        "campaignId": {
+            "type": "integer"
+        },
+        "headingLevel": {
+            "type": "number",
+            "default": 1
+        },
+        "textAlign": {
+            "type": "string"
```

Comment:
Would it be interesting to add some default value for this attribute? E.g: `"default": "left"`